src/Module/WellKnown/NodeInfo.php

```diff
@@ -39,6 +39,10 @@ protected function rawContent(array $request = [])
 				 'href' => DI::baseUrl() . '/nodeinfo/1.0'],
 				['rel'  => 'http://nodeinfo.diaspora.software/ns/schema/2.0',
 				 'href' => DI::baseUrl() . '/nodeinfo/2.0'],
+				 ['rel'  => 'http://nodeinfo.diaspora.software/ns/schema/2.1',
+				 'href' => DI::baseUrl() . '/nodeinfo/2.1'],
+				 ['rel'  => 'http://nodeinfo.diaspora.software/ns/schema/2.2',
```

Comment:
These lines don't match the original alignment, please fix either the first two or the last two to normalize the indentation.

Reply:
It's now normalized.